### DISCUSSION TITLE
关于VisualGLM训练的各种Tricks

### DISCUSSION THREAD

**Discussion by WangRongsheng** [WangRongsheng/XrayGLM#5] [General]
如果数据多的话可以考虑增加训练的参数，目前的训练脚本只训练了2层lora：https://github.com/THUDM/VisualGLM-6B/issues/61#issuecomment-1562856845

**Comment by WangRongsheng** [comment]
大模型训练中，训练的精度（int4，int8，float16...）不重要，模型的参数（6B、13B...）很影响模型的性能。

**Comment by freelancerllm** [comment]
目前glm系列的模型目前就6B。

**Reply by WangRongsheng** [reply]
> 目前glm系列的模型目前就6B。

是的，希望后面会有一些大参数的模型出来。

**Comment by WangRongsheng** [comment]
使用QLoRA最低显存要求9.8G，可以多卡，可以将脚本里的--include localhost:0改成你想使用的卡，比如--include localhost:0,1,2，或者直接去掉这个参数，就是使用所有卡了：https://github.com/THUDM/VisualGLM-6B/commit/5d368f62da71417c9685ecff8fe55f1a60ce8784#commitcomment-115614063

**Comment by WangRongsheng** [comment]
![3](https://github.com/WangRongsheng/XrayGLM/assets/55651568/91f09cbd-1076-4d33-b8a9-f1c602d7db38)

**Comment by WangRongsheng** [comment]
支持多轮对话finetune：https://github.com/THUDM/VisualGLM-6B/issues/118

**Comment by WangRongsheng** [comment]
关于微调图片的大小问题：https://github.com/THUDM/VisualGLM-6B/issues/82
